### ISSUE TITLE
Dashboard checkbox

### ISSUE THREAD

**Issue by Remington-Hunter** [thingsboard/thingsboard#11698]
How can I add a checkbox function to the entites table in the dashboard? Can I only modify the source code to add it? Can I customize the widget to achieve this function?

**Comment by devdoob** [comment]
Yes you have to write a custom widget to achieve this multiple selection in table rows

**Comment by Remington-Hunter** [comment]
Hello, where can I get some examples of custom widgets? I want to add a checkbox based on the entities table in the dashboard. Can this be achieved through customization? Or do I need to modify the source code to achieve this?

**Comment by Remington-Hunter** [comment]
The reason I want to add a checkbox is that I want to send rpc commands to devices in batches to control them. My current approach is to customize a dialog box and then enter multiple devices through a multi-select input box to send rpc commands in batches. I want to add this checkbox to control devices more easily.

**Comment by txf-** [comment]
Sorry for bumping this.

 I have added a similar checked unchecked function by double clicking a row. I am using browser local storage to maintain a list of checked items, to avoid calling apis for every single row.

However there is still some latency between "checking" or "unchecking" the row and when the row changes color. Is there a way to reduce this latency?

**Comment by mykhailokornieiev** [comment]
As the community has not provided assistance on this matter over an extended period, I am closing this ticket due to inactivity. If it still relevant, feel free to open a new ticket with full description of your request.
